.github/workflows/edge.yml

```diff
@@ -0,0 +1,64 @@
+name: Mondoo Docker Tag Edge
+
+on:
+  workflow_dispatch:
```

Comment:
The goal would be to trigger this workflow from the Mondoo client release just after the binaries have been pushed